docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -232,6 +232,10 @@ like so::
 
    $ pip install -r tests/requirements/py3.txt  # Python 2: py2.txt
 
+If you are experiencing any problems during the installation of these 
+dependencies, please see our :ref:`troubleshooting <troubleshooting-unit-testing>` 
+page.
```

Comment:
I haven't looked at the docs here in full to recommend that it go in Miscellaneous _per se_, but since dependency issues affect more than just Django developers running Django's own unit test suite, it definitely _doesn't_ make sense for solutions to a more general problem to live under Unit Testing Problems. Does that make sense?

Miscellaneous would definitely be better than Unit Testing Problems, though. Maybe it calls for specifically having its own section? Perhaps "External Dependencies"?